### ISSUE TITLE
[Bug]: MorePeripherals requires update to 1.7.2 for grinder bugfix

### ISSUE THREAD

**Issue by 1480c1** [FTBTeam/FTB-Modpack-Issues#1360]
### Modpack

FTB Plexiglass mountain

### Modpack version

1.2.0

### Describe the bug

upstream issue: https://github.com/JheffersonMarques/MorePeripherals/issues/12

Copying to relevant fields

I can confirm that manually updating to 1.7.2 seem to fix the issue (checked the grinder for more than 10 seconds, and it did not disappear)

### Steps to reproduce

1. Place down Grinder
2. Place wooden sword in Grinder and keep the GUI open
3. Observe it disappearing between 0-3 seconds

### Expected behaviour

Grinder doesn't eat the sword

### Additional information

https://user-images.githubusercontent.com/8345542/193141674-1e242c50-a649-4afd-ab9a-0afdfa3c267c.mp4

**Comment by 1480c1** [comment]
Seems there's an additional bug when trying to use the grinder on a server. Might be best to hold off until that's fixed

**Comment by Raidobw2** [comment]
Thank you very much for forwarding it to the author and giving them that much information!

**Comment by 1480c1** [comment]
Upstream pushed a fix to 1.7.3

**Comment by 1480c1** [comment]
Tested 1.7.3 and can confirm it works

**Comment by Raidobw2** [comment]
Awesome, thank you for your help!

**Comment by Raidobw2** [comment]
Yes, apologies, the StoneBlock 3 launch got me quite busy helping the lead modpack developer. Thanks again for your help on this issue and for reminding me after! I appreciate it!
More updates are to come :)

**Comment by 1480c1** [comment]
I am looking forward to playing stoneblock 3 sometime, good luck with your dev work

**Comment by Raidobw2** [comment]
I hope you'll enjoy it and thank you!
